vincechain/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "vincechain",
+  "assets": [
+    {
+      "description": "Vince Chain is an Industry-Grade Blockchain For DeFi, Remote work, Gaming, Web3 and will power the new internet.",
+      "denom_units": [
+        {
+          "denom": "avce",
+          "exponent": 0
+        },
+        {
+          "denom": "vce",
+          "exponent": 18
+        }
+      ],
+      "base": "avce",
+      "name": "VinceChain",
+      "display": "vincechain",
```

Comment:
refers to the denom unit to display as. Please correct to 'vce'

Reply:
this value for 'display' cannot be 'vincechain' because 'vincechain' is not defined as a denom_unit.